pkg/secrets/secrets_test.go

```diff
@@ -20,22 +19,3 @@ func TestGetLambdatestSecrets(t *testing.T) {
 	lambdatestSecrets := secretsManager.GetLambdatestSecrets()
 	assert.Equal(t, "dummysecretkey", lambdatestSecrets.SecretKey)
 }
-
-func TestWriteGitSecrets(t *testing.T) {
```

Comment:
If we are removing test , can we write test for there counter part logic , basically the functionality which writes the git secret should be tested too

Reply:
Added test case.